operations/deployment/terraform/security-group.tf

```diff
@@ -36,9 +36,9 @@ resource "aws_security_group_rule" "ingress_ssh" {
 resource "aws_security_group_rule" "ingress_lb_port" {
   count       = var.lb_port != var.app_port ? 1 : 0 
   type        = "ingress"
-  description = "${var.aws_resource_identifier} - Port"
-  from_port   = tonumber(var.lb_port != "" ? var.lb_port : ( local.cert_available ?  443 : 80 ) )
-  to_port     = tonumber(var.lb_port != "" ? var.lb_port : ( local.cert_available ?  443 : 80 ) )
+  description = "${var.aws_resource_identifier} - lb Port"
+  from_port   = tonumber(var.lb_port != "" ? var.lb_port : 443 )
+  to_port     = tonumber(var.lb_port != "" ? var.lb_port : 443 )
```

Comment:
wouldn't this have issues if someone didn't specify an LB port and doesn't have cert available?  in that case, it probably shouldn't be 443, right?

Reply:
Context for the change;
https://bitovi.slack.com/archives/C0129U3GL93/p1674669607224819

I agree that 443 is not the correct port, however the elb is pointing at var.lb_port and defaulting to 443. 

The `80` possible route was causing conflict issues with the above security group rule which was already created a target to 80 (causing a dup issue)

I mentioned in the Slack thread that this was a work around and that I wanted to speak to Leo to understand his thinking

Reply:
Found the issue and fixed 👍 
 Reverted the above section so I'm resolving this conversation